docs/reference/mapping/types/numeric.asciidoc

```diff
@@ -173,9 +175,16 @@ support this parameter.
     the <<mapping-source-field,`_source`>> field. Accepts `true` or `false`
     (default).
 
-<<mapping-field-meta,`meta`>>::
-
-    Metadata about the field.
+// tag::time_series_metric[]
+`time_series_metric`::
+For internal use by Elastic only. Marks the field as a time series metric. The
+value is the metric type. Defaults to `null` (Not a time series metric).
+// end::time_series_metric[]
++
+For numeric fields, this parameter accepts `gauge` and `counter`. You can't
+update this parameter for existing fields.
++
+`scaled_float` and `unsigned_long` fields don't support this parameter.
```

Comment:
I am not sure where does the constraint that `scaled_float` and `unsigned_long` fields don't support this parameter. There is no such constraint in the code. Am I missing something here?

Reply:
Thank you for opening the issue.  I will check that asap.